docs/examples/websockets/dependency_injection_yield.py

```diff
@@ -0,0 +1,27 @@
+from typing import TypedDict
```

Comment:
can you rebase this PR on the litestar stuff?

Reply:
I did.